app/README.md

```diff
@@ -106,3 +106,30 @@ We did not follow their [install directions](https://designsystem.digital.gov/do
 Compiling the USWDS sass is slow, so the initial build step and subsequent sass re-compiles are slow, but after the design system is set up, we shouldn't need to be regularly re-compiling sass.
 
 Copying the USWDS static assets into the project is handled by a [yarn postinstall](https://classic.yarnpkg.com/lang/en/docs/package-json/#toc-scripts) script in `package.json`.
+
+## Internationalization (i18n)
+
+### Next.js i18n
+
+We are using [next-i18next](https://github.com/i18next/next-i18next) for Next.js internationalization. It provides a Next.js wrapper around [i18next](https://www.i18next.com/) and [react-i18next](https://github.com/i18next/react-i18next). Configuration is located in `next-i18next.config.js`. To add a language:
+
+1. Edit `next-i18next.config.js` and add the language to `locales`
+2. Add a language folder: `mkdir -p public/locales/<lang>`
+3. Add a language file: `touch public/locales/<lang>/common.json` and add the translated content
```

Comment:
ah, I didn't realize we'd put i18n content in the `public/` directory. You can ignore my comment on your other PR about ignoring all of `public/` from Prettier then. 

I'm wondering how this impacts JS bundles and compression – do these files get minified when they're in the `public/` directory, which I think is set as our static directory?